cli/vm/cli.go

```diff
@@ -180,6 +180,17 @@ both parameters are mandatory, example:
 > loadtx /path/to/file`,
 		Action: handleLoadTx,
 	},
+	{
+		Name:      "loaddeployed",
+		Usage:     "Load deployed contract into the VM from chain",
+		UsageText: `loaddeployed [--historic <height>] <hash-or-address-or-id>`,
```

Comment:
Yes! Because it uses interop context to get the contract state, and the contract itself is stored inside management storage. The historic state is properly set up by `--historic` flag. The only thing left is working with destroyed contracts by ID, but even this can easily be done after https://github.com/nspcc-dev/neo-go/pull/2702.

Reply:
Fixed, but it still can be improved after #2702 merge.